### PR TITLE
ida plugin: update documentation

### PR DESCRIPTION
**Pull request by mike-hunhoff**
updates IDA plugin documentation.

### CODE REVIEW

**Comment by williballenthin** [comment]
image
this file probably becomes the largest file in the repo by far. though, at 3mb, its still not that big.

File: capa/ida/plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+* Display rule source content when a user hovers their cursor over a rule match
+* Double-click `Address` column to view associated feature in the IDA Disassembly view
+* Limit tree view results to the function currently displayed in the IDA Disassembly view; update results as a user navigates to different functions
+* Sort results by column
+* Reset tree view and IDA Disassembly view highlighting by clicking `Reset`
+* Automatically re-analyze database when user performs a program rebase
+* Automatically update results when IDA is used to rename a function
+* Select one or more checkboxes to highlight the associated addresses in the IDA Disassembly view
+* Right-click a function match to rename it; the new function name is propagated to the current IDA database
+* Right-click to copy a result by column or by row
```
Comment (williballenthin) [inline capa/ida/plugin/README.md:35]: this list is rather long. i think its a reasonable compromise between authoring a bunch of text that few people will read closely and not providing enough detail. this is enough to get started (and excited).

to emphasize the most important/interesting elements, put them first.

File: capa/ida/plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+### Installation
+
+You can install capa explorer using the following steps:
+
+1. Install capa for the Python interpreter used by your IDA installation:
+    ```
+    $ pip install flare-capa
+    ```
+3. Copy [capa_explorer.py](https://raw.githubusercontent.com/fireeye/capa/master/capa/ida/plugin/capa_explorer.py) to your IDA plugins directory
```
Comment (williballenthin) [inline capa/ida/plugin/README.md:62]: numbers here go from 1 -> 3

Comment (williballenthin) [reply capa/ida/plugin/README.md:62]: also, should we reference the rules download here?

File: capa/ida/plugin/README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+### Usage
+
+1. Run IDA and analyze a supported file type (select `Manual Load` and `Load Resources` for best results)
+2. Open capa explorer in IDA by navigating to `Edit > Plugins > FLARE capa explorer` or using the keyboard shortcut `Alt+F5`
+3. Click `Analyze`
+
+When running capa explorer for the first time you are prompted to select a file directory containing capa rules. The plugin conveniently
+remembers your selection for future runs; you can change this selection by navigating to `Rules > Change rules directory...`. We recommend 
+downloading and using the [standard collection of capa rules](https://github.com/fireeye/capa-rules) when getting started with the plugin.
```
Comment (williballenthin) [inline capa/ida/plugin/README.md:72]: i like this explanation and wording.

**Review by williballenthin — APPROVED** [review]
looks great!

**Review by mr-tz — APPROVED** [review]
great job, Mike!

File: capa/ida/plugin/README.md (hunk 2 above)
Comment (mike-hunhoff) [reply capa/ida/plugin/README.md:62]: Good point! I'll copy the rules download from the usage section here.

And numbering from 1 -> 3 was the only way I could get the correct numbers to render 🤷‍♂️:

Screen Shot 2020-09-14 at 11 32 15 AM

**Comment by mike-hunhoff** [comment]
> image
> this file probably becomes the largest file in the repo by far. though, at 3mb, its still not that big.

Yeah, I really worked to get this down to a reasonable size, but 3.3 MB was the best I could do because of it's length. We can always remove this in the future if the size becomes an issue.

**Pull request merged**